src/controllers/IframeController.tsx

```diff
@@ -17,7 +17,7 @@ export function IframeController({ currentConfig, provState, answers }: { curren
   const dispatch = useDispatch();
   const identifier = useCurrentIdentifier();
   const [height, setHeight] = useState(800);
-  const iframeStyle = useMemo(() => ({ height: `${height}px`, width: '100%' }), [height]);
+  const iframeStyle = useMemo(() => ({ height: `${height}px`, width: '100%', border: 0 }), [height]);
```

Comment:
This will conflict with my changes. Let's be careful about merge order and making sure we don't lose this in the conflicts